src/python/WMCore/ACDC/CouchFileset.py

```diff
@@ -99,9 +99,22 @@ def add(self, files, mask = None):
         _add_
 
         Add files to this fileset
+
+        Note: if job was lumi based splitted, then we do not have
+        reliable events information. If job was event based splitted,
+        then we do not have reliable lumi information.
         """
         filteredFiles = []
         if mask:
+            for f in files:
+                # There is no LastEvent for last job of a file
+                if mask['LastEvent'] and mask['FirstEvent']:
+                    f['events'] = mask['LastEvent'] - mask['FirstEvent']
```

Comment:
Yes, inclusivemask was True for all my tests. Good catch!
Let me update this code.